frontend/js/src/playlists/Playlist.tsx

```diff
@@ -319,8 +320,29 @@ export default function PlaylistPage() {
 
   return (
     <div role="main">
-      <Helmet>
-        <title>{playlist.title} - Playlist</title>
+      <Helmet defer={false}>
+        <title>
```

Comment:
What does defer here do? I was reading through the available documentations, but couldn't find it there.

Reply:
Left over from my previous (failed) attempt at getting the meta tags to be readable directly at page load, will remove.

> {/* (optional) set to false to not use requestAnimationFrame and instead update the DOM as soon as possible.
        Useful if you want to update the title when the tab is out of focus 
    */}
    defer={false}